src/styles/vars.scss

```diff
@@ -41,18 +41,18 @@
   --color-primary: var(--hopbush);
   --color-secondary: var(--bouquet);
   --color-accent: var(--patina);
-  --color-shadow: rgba(var(--black), 0.125);
+  --color-shadow: rgb(var(--black) 0.125);
   --color-note: #666;
 
   // Common colors
   --color-background: var(--primary-white);
   --color-background-shade: var(--iron);
   --color-background-invert: var(--hopbush);
-  --color-text: darken(var(--pale-sky), 15);
```

Comment:
it was some mystery moment. css var could not interfere with scss function. It was broken :(